.lintrunner.toml

```diff
@@ -1552,7 +1552,7 @@ init_command = [
 ]
 is_formatter = true
 
-# This linter prevents merge conlicts in csv files in pytorch by enforcing
+# This linter prevents merge conflicts in csv files in pytorch by enforcing
```

Comment:
Could be done in separate PR (or should I say that it conlicts with PR title :) )